devices/XPT2046/XPT2046.cs

```diff
@@ -0,0 +1,202 @@
+﻿// Copyright (c) .NET Foundation and Contributors
+// Portions Copyright (c) Microsoft Corporation.  All rights reserved.
+// See LICENSE file in the project root for full license information.
+
+using System;
+using System.Device.Spi;
+using System.Diagnostics;
+using System.IO;
+
+namespace Iot.Device.Xpt2046
+{
+    /// <summary>
+    /// Class representing the XPT2046 device driver.
+    /// </summary>
+    public class Xpt2046 : IDisposable
+    {
+        private const byte StartBit = 0b10000000;
+        private const int MaxConversion = 0b111111111111;
+
+        private readonly SpiDevice _spiDevice;
+        private readonly int _xMax;
+        private readonly int _yMax;
+        private bool _disposedValue = false;
+
+        private enum PowerMode
+        {
+            PowerDown = 0b00000000,              // Power Down between conversions
```

Comment:
it's minimal: we're putting the comments on top of the line. I saw there are more like this over the file.